frontend/modules/sequences/sequences.selectors.ts

```diff
@@ -24,9 +24,9 @@ import { getDateByStep, getSelectedFrame } from './sequences.helper';
 
 export const selectSequencesDomain = (state) => (state.sequences);
 
-const getMinMaxDates = ({frames}) => ({
-	minDate: (frames[0] || {}).dateTime,
-	maxDate: (frames[frames.length - 1] || {}).dateTime
+const getMinMaxDates = ({frames, minDate, maxDate}) => ({
```

Comment:
with the change they should always have a min/max dates now?

Does this function need to exist?

it's very odd to be passing in the result just to return it.

Reply:
also can we rename this to endDate and startDate?